.github/workflows/ci.yml

```diff
@@ -6,52 +6,25 @@ on:
     branches:
       - main
 
-env:
-  RUST_BACKTRACE: 1
-  CARGO_TERM_COLOR: always
-
 jobs:
+  compare-dist:
+    name: Check if compiled action in dist/ matches the source files
+    uses: ./.github/workflows/compare-dist.yml
+
```

Comment:
Can we add a TypeScript linter pass as well? Linters and auto-formatters are best added as early as possible — less lint we'll have to fix down the line.

I'm not an expert on the subject but the advice in these links looks credible and reasonably up-to-date:
https://blog.logrocket.com/linting-typescript-eslint-prettier/
https://blog.logrocket.com/using-prettier-eslint-automate-formatting-fixing-javascript/

Reply:
You're right, these are definitely must-haves! I've added both ESLint and Prettier checks.